docs/source/quicktour.mdx

```diff
@@ -32,18 +32,33 @@ lighteval accelerate \
      "leaderboard|truthfulqa:mc|0|0"
 ```
 
-Here, `--tasks` refers to either a comma-separated list of supported tasks from
-the [tasks_list](available-tasks) in the format:
+Here, the first argument specifies which model(s) to run, and the second argument specifies how to evaluate them.
+
+Multiple models can be evaluated at the same time by using a comma-separated list. For example:
```

Comment:
Nope, we can only evaluate one model at a time - however we can specifiy precision, peft weights, ...